packages/clay-multi-select/src/__tests__/index.tsx

```diff
@@ -252,19 +244,8 @@ describe('Interactions', () => {
 			/>
 		);
 
-		expect(document.querySelectorAll('.dropdown-item').length).toEqual(0);
-
 		expect(onItemsChangeFn).not.toHaveBeenCalled();
 
-		fireEvent.change(
-			document.querySelectorAll(
-				'input[type=text]'
-			)[0] as HTMLInputElement,
-			{target: {value: 'bar'}}
-		);
-
-		expect(document.querySelectorAll('.dropdown-item').length).toEqual(1);
-
```

Comment:
These were removed because they had nothing to do with the purpose of the test, making sure the onItemsChange was called.

Reply:
Makes sense, thanks